src/Compilers/VisualBasic/Portable/Binding/Binder.vb

```diff
@@ -503,7 +503,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
 
             Dim emittedName = MetadataTypeName.FromFullName(metadataName, useCLSCompliantNameArityEncoding:=True)
-            Return Me.ContainingModule.LookupTopLevelMetadataType(emittedName)
+            Dim result As NamedTypeSymbol = Me.ContainingModule.LookupTopLevelMetadataType(emittedName)
+            Debug.Assert(If(Not result?.IsErrorType(), True))
```

Comment:
Is there a reason we can't just do `result?.IsErrorType() IsNot True`? I find the coalescing operator hard to read here.

Reply:
An `Is` or `IsNot` expression is used to do a reference equality comparison.

Reply:
Is there a non-coalescing version of this that we can use then?

Reply:
> Is there a non-coalescing version of this that we can use then?

The other forms I considered:
- ```Not ((result?.IsErrorType()).GetValueOrDefault())```
- ```result is Nothing OrElse Not result.IsErrorType()```

Reply:
Alright. I don't think any of those are enough of an improvement to insist on it.